pre-index.php

```diff
@@ -148,6 +148,8 @@ function pokemonFilterImages($noPokemonNumbers)
         </a>';
         }
         ?>
+        <a href="#stats" id="statsToggle" class="statsNav" style="float: right;"><span
```

Comment:
This is probably a merge conflict, now Stats is double in navbar because of this